uwblueprint/focus-on-nature
# Migrate OAuth login to new package

<img width="456" alt="Screen Shot 2022-12-27 at 1 53 29 AM" src="https://user-images.githubusercontent.com/40974372/209648530-61b13b76-1b83-421a-84a3-08a12b0b4fb1.png">



## Implementation description
* Using this new package: https://www.npmjs.com/package/@react-oauth/google
  * Sample - https://github.com/MomenSherif/react-oauth/blob/47ec50f7f6b32d96f20cac63b21533ae74b49e3b/apps/playground/src/components/SignInFlows.tsx#L40-L47
  * Styles - https://github.com/MomenSherif/react-oauth/blob/8ed49b8ef6cd6f59fd11c233fe8157a381413076/packages/%40react-oauth/google/src/types/index.ts#L70 
* https://developers.google.com/identity/gsi/web/guides/migration



## Steps to test
1. Log in with bp email
2. See error state from logging in with non-bp email



## What should reviewers focus on?
* It's not the right colour, but the styles didn't have it as an option. Didn't think it was worth building completely from scratch

<img width="1437" alt="Screen Shot 2022-12-27 at 1 57 11 AM" src="https://user-images.githubusercontent.com/40974372/209649014-056e6a20-a78d-4c69-8f9d-eb1c03f6725b.png">

<img width="1437" alt="Screen Shot 2022-12-27 at 1 57 50 AM" src="https://user-images.githubusercontent.com/40974372/209649108-50260500-b546-4085-b3c8-5b2c99da53d1.png">

## Checklist
